bundles/org.openhab.binding.onecta/src/main/resources/OH-INF/thing/onecta-climatecontrol.xml

```diff
@@ -0,0 +1,1807 @@
+			<channel id="energy-heating-month-1" typeId="acunit-energy-heating-month-1"/>
+			<channel id="energy-heating-month-2" typeId="acunit-energy-heating-month-2"/>
+			<channel id="energy-heating-month-3" typeId="acunit-energy-heating-month-3"/>
+			<channel id="energy-heating-month-4" typeId="acunit-energy-heating-month-4"/>
+			<channel id="energy-heating-month-5" typeId="acunit-energy-heating-month-5"/>
+			<channel id="energy-heating-month-6" typeId="acunit-energy-heating-month-6"/>
+			<channel id="energy-heating-month-7" typeId="acunit-energy-heating-month-7"/>
+			<channel id="energy-heating-month-8" typeId="acunit-energy-heating-month-8"/>
+			<channel id="energy-heating-month-9" typeId="acunit-energy-heating-month-9"/>
+			<channel id="energy-heating-month-10" typeId="acunit-energy-heating-month-10"/>
+			<channel id="energy-heating-month-11" typeId="acunit-energy-heating-month-11"/>
+			<channel id="energy-heating-month-12" typeId="acunit-energy-heating-month-12"/>
+			<channel id="energy-heating-month-13" typeId="acunit-energy-heating-month-13"/>
+			<channel id="energy-heating-month-14" typeId="acunit-energy-heating-month-14"/>
+			<channel id="energy-heating-month-15" typeId="acunit-energy-heating-month-15"/>
+			<channel id="energy-heating-month-16" typeId="acunit-energy-heating-month-16"/>
+			<channel id="energy-heating-month-17" typeId="acunit-energy-heating-month-17"/>
+			<channel id="energy-heating-month-18" typeId="acunit-energy-heating-month-18"/>
+			<channel id="energy-heating-month-19" typeId="acunit-energy-heating-month-19"/>
+			<channel id="energy-heating-month-20" typeId="acunit-energy-heating-month-20"/>
+			<channel id="energy-heating-month-21" typeId="acunit-energy-heating-month-21"/>
+			<channel id="energy-heating-month-22" typeId="acunit-energy-heating-month-22"/>
+			<channel id="energy-heating-month-23" typeId="acunit-energy-heating-month-23"/>
+
+			<channel id="energy-heating-current-day" typeId="acunit-energy-heating-current-day"/>
+			<channel id="energy-heating-current-year" typeId="acunit-energy-heating-current-year"/>
+
+		</channels>
+	</channel-group-type>
+
+
+	<channel-type id="acunit-energy-heating-month-0" advanced="true">
+		<item-type>Number:Energy</item-type>
+		<label>Energy heating last year (00) January</label>
+		<description>The energy usage for heating last year January</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="acunit-energy-heating-month-1" advanced="true">
+		<item-type>Number:Energy</item-type>
+		<label>Energy heating last year (01) February</label>
+		<description>The energy usage for heating last year February</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="acunit-energy-heating-month-2" advanced="true">
+		<item-type>Number:Energy</item-type>
```

Comment:
without providing a unitHint, the default unit will be used, is that expected? 
Applies to all UoM channels